src/webhook/mutation/pod_mutator/register.go

```diff
@@ -49,11 +49,17 @@ func registerInjectEndpoint(mgr manager.Manager, webhookNamespace string, webhoo
 		return errors.WithStack(err)
 	}
 
+	deployedViaOLM, err := kubesystem.IsDeployedViaOlm(apiReader, webhookPodName, webhookNamespace)
+	if err != nil {
+		return errors.WithStack(err)
```

Comment:
Error is internal, must not be wrapped
```suggestion
		return err
```

Reply:
True, also removed it for return values above